includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -433,8 +435,46 @@ public function getData() {
 		return $data;
 	}
 
+	private function getSitenameData() {
+		$sitename = $this->wg->Sitename;
+		$wgSitenameForComscoreForWikiId = WikiFactory::getVarValueByName( 'wgSitenameForComscore', $this->wikiId );
+
+		if ( $wgSitenameForComscoreForWikiId ) {
+			$sitename = $wgSitenameForComscoreForWikiId;
+		} else {
+			$wgSitenameForWikiId = WikiFactory::getVarValueByName( 'wgSitename', $this->wikiId );
+
+			if ( $wgSitenameForWikiId ) {
+				$sitename = $wgSitenameForWikiId;
+			}
+		}
+
+		return [
+			'type' => 'text',
+			'value' => $sitename
```

Comment:
What do you thin about?
$sitename ?? $this->wg->Sitename

I think the flow here would be cleaner